src/api/app/models/package.rb

```diff
@@ -151,13 +151,23 @@ def self.internal_get_project(project)
     prj
   end
 
+  def self.normalize_package_name(name)
+    # exception for package names used to have a collon
```

Comment:
Because we have them? product containers did always contain a colon and this is according to our verification code since SLE 11 code ....